go/interpreter/lfvm/instructions.go

```diff
@@ -1080,6 +1077,19 @@ func genericCall(c *context, kind tosca.CallKind) {
 	}
 
 	baseGas := c.memory.ExpansionCosts(needed_memory_size)
+
+	accessCost := tosca.Gas(0)
+	if c.isAtLeast(tosca.R09_Berlin) {
+		// Check address presence in the access list
+		isWarm := c.context.AccessAccount(toAddr)
+		// The WarmStorageReadCostEIP2929 (100) is already deducted in the form of a constant cost, so
+		// the cost to charge for cold access, if any, is Cold - Warm
+		if !isWarm {
+			accessCost = ColdAccountAccessCostEIP2929 - WarmStorageReadCostEIP2929
```

Comment:
If those constants have only been used in this context, consider moving them into this function. If they are only used in this file, move them into this file.

Reply:
Ok.